### DISCUSSION TITLE
Unit 4.4 Captions

### DISCUSSION THREAD

**Discussion by gracecluvohio** [Lightning-AI/dl-fundamentals#79] [Lecture videos and quizzes] [answered]
I'm unsure if this is an issue on my side, but I believe the captions for the unit 4.4 part 3 video are incorrect. They seem to be from another video.

**Comment by rasbt** [comment, marked as answer]
Thanks! I'll look into it!

**Reply by rasbt** [reply]
Should be fixed now! Thanks again!
